tests/Utility/HttpStatus/GetTextTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace WpOrg\Requests\Tests\Utility\HttpStatus;
+
+use WpOrg\Requests\Exception;
+use WpOrg\Requests\Exception\InvalidArgument;
+use WpOrg\Requests\Tests\TestCase;
+use WpOrg\Requests\Tests\TypeProviderHelper;
+use WpOrg\Requests\Utility\HttpStatus;
+
+/**
+ * @coversDefaultClass \WpOrg\Requests\Utility\HttpStatus
+ */
+class GetTextTest extends TestCase {
+	/**
```

Comment:
Nitpick: could we have the data provider methods _after_ (directly following) the test methods ? That would improve consistency with the other (already revised) test classes.

(same for the other test class)